### DISCUSSION TITLE
Where can I learn more about MarkovJunior's probabilistic inference mechanism?

### DISCUSSION THREAD

**Discussion by gbrlfaria** [mxgmn/MarkovJunior#23] [Q&A]
Hello,

As the title states, where can I learn more about how MarkovJunior performs probabilistic inference? Any good resources on the algorithms and concepts involved would be appreciated.

In the future, could the author consider writing a blog post giving a high-level overview of the inference mechanism? I'd certainly appreciate that as well.

**Comment by mxgmn** [comment]
Hi! Will do.
